eo-runtime/src/main/eo/org/eolang/array.eo

```diff
@@ -26,14 +26,7 @@
 +rt jvm org.eolang:eo-runtime:0.0.0
 +version 0.0.0
 
-# @todo #1060:30min The method .empty is not nessesary.
-#  Make the create of array possible just by using
-#  the object name, i.e `array > a` and remove this
-#  method.
 [] > array
-  # Create empty array
-  [] > empty
```

Comment:
@andreoss I think we should just delete this method and that's it. We can't do the changes you made, since they break the principles of EO. An object can be created _only_ by copying an abstract object _and_ providing parameters.

Reply:
@yegor256 Any suggestion how `array.with 1` should be interpreted than?
`array` could be a keyword which would be treated as `(*)`?

Reply:
@andreoss I'm wrong about deleting `array.empty`. This is what I think we should have: `array.with` should be an atom, which internally makes a new array and creates a new `array` object and return it. `array` should not be a data object, but a regular one. `array.empty` should be an atom that makes a new `array` object with empty array inside.